# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wpa_supplicant/prefix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- wpa_supplicant-0.6.0/wpa_supplicant/Makefile.vanilla	2007-06-10 14:44:52.081832235 +0200
+++ wpa_supplicant-0.6.0/wpa_supplicant/Makefile	2007-06-10 14:45:08.058742707 +0200
@@ -6,6 +6,8 @@
 CFLAGS = -MMD -O2 -Wall -g
 endif
 
+prefix=/usr/local
+
 CFLAGS += -I.
 CFLAGS += -I../src
 CFLAGS += -I../src/crypto
@@ -32,8 +34,8 @@
 	echo CONFIG_WIRELESS_EXTENSION=y >> .config
 
 install: all
-	mkdir -p $(DESTDIR)/usr/local/sbin/
-	for i in $(ALL); do cp $$i $(DESTDIR)/usr/local/sbin/$$i; done
+	mkdir -p $(DESTDIR)$(prefix)/sbin/
+	for i in $(ALL); do cp $$i $(DESTDIR)$(prefix)/sbin/$$i; done
 
 OBJS = config.o \
 	common.o md5.o md4.o \
